### ISSUE TITLE
System to itemize JF Transfer regardless of amount

### ISSUE THREAD

**Issue by AureliaKhorsand** [fecgov/fecfile-web-app#486]
Business Reason

Per FECA regulations, all Form 3X transactions on Line 12 must be itemized on Schedule A, regardless of amount. This includes JF Transfer transactions.

Acceptance Criteria

***Given*** one of the following transactions:

*   Joint Fundraising Transfer
*   PAC JF Transfer Memo

***Then*** the transaction will be itemized upon saving

 ***and*** appear on it's own line item on the *.fec file

QA notes

1\. In a report, make a JF Transfer transaction. (to make a later step easier, i'd recommend putting something in the memo text that you cans search for like "jf transfer test 1"

 2\. go to the reports list page. find the report where you added the JF Transfer and click the second green button on that row (for some reason the icons are missing)

 3\. you should be on a page that says "Test .FEC generation for "

 4\. Click "generate .fec". another button will appear, but wait about 5 seconds before pressing it

 5\. Click "download .fec" after about 5 seconds

 6\. Open the file that was just downloaded with notepad for PC or textedit for Mac. There must be a row representing the JF Transfer that was made in step 1, regardless of contribution aggregate (other transaction types show up depending on if the contribution aggregate is above $200). You can be sure the line for the transaction you made if the text at the end of the line matches the memo text you filled out in the form.

### QA Notes ###
null

### DEV Notes ###
null

### Design ###
null

 See full ticket and images here: [FECFILE-1077](https://fecgov.atlassian.net/browse/FECFILE-1077)

**Comment by Elaine-Krauss-TCG** [comment]
If I create a report with  JF Transfer and PAC JF Transfer Memo receipts,
![image.png](https://images.zenhubusercontent.com/626ab5e81394ace64f8ed65d/e652cb6a-735f-42ab-a0f1-eea89ffa8fcc)

then when I generate a .fec
![image.png](https://images.zenhubusercontent.com/626ab5e81394ace64f8ed65d/65f939d0-b740-49fc-821c-5f1122fe1ff0)

The .fec has itemized each of those transactions
![image.png](https://images.zenhubusercontent.com/626ab5e81394ace64f8ed65d/9ec3f757-216d-4aa1-93cb-475b088d59fc)

It's a little hard to parse, visually speaking, but you can see that the two Schedule A, Line 12 transactions are present with the same uuid's and contribution amounts.

**Comment by Elaine-Krauss-TCG** [comment]
Side note: downloading the generated .fec files seems to be broken on DEV at the moment.  This was tested by manually retrieving a .fec that was generated on local.

**Comment by WiseQA** [comment]
QA inspection.   No Code Review was completed on this ticket moved back to code review @mjtravers @MitchellTCG @Elaine-Krauss-TCG

**Comment by mjtravers** [comment]
Passes CR, ready for QA

**Comment by WiseQA** [comment]
QA inspection review of the output (see screenshot below) per DEV.  This was also examined and verified during CR of expected results.

![image.png](https://images.zenhubusercontent.com/61ba01e428a658b4eb0ca758/859a66e0-2b53-4d28-8997-025ae53d4670)

QA Inspection Review Completed.   Moved to Stage Ready.
